source/DiscordManager.hx

```diff
@@ -0,0 +1,57 @@
+package ;
+
+import flixel.util.FlxTimer;
+import discord_rpc.DiscordRpc;
+
+class DiscordManager
+{
+    public static function init() {
+        DiscordRpc.start({
+            clientID : "806658783132516383",
+            onReady : _onReady,
+            onError : _onError,
+            onDisconnected : _onDisconnected
+        });
+
+        DiscordRpc.process();
+        new FlxTimer().start(1, process, 0);
+
+        trace("Discord RPC Initialized!");
+    }
+
+    static function process(timer:FlxTimer):Void {
+        DiscordRpc.process();
+    }
+
+    static function _onReady() {
+        DiscordRpc.presence({
+            state : "Title Screen",
+            largeImageKey : "icon",
+            largeImageText : "Friday Night Funkin'"
```

Comment:
For reference, this is what it looks like on discord's end
![](https://i.imgur.com/slfhF7H.png)